addons/web/static/src/views/graph/graph_model.js

```diff
@@ -243,6 +243,7 @@ export class GraphModel extends Model {
                     domains: new Array(dataLength).fill([]),
                     label: datasetLabel,
                     originIndex: originIndex,
+                    identifier: identifier,
```

Comment:
`identifier,`

Reply:
don't think it is good, two data points with different identifiers can have same datasetLabel

Reply:
@Polymorphe57 Hello again, i wasn't able to finish this PR in the last few months sorry :)
Could you tell me if my new fix is accurate?
Thanks for the review